### PR TITLE
menu list unit tests

### PR DESCRIPTION
**Pull request by brianchristopherbrady**
Adds MenuList playwright tests

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8c3c3b2fbc9cf08609ae30d5f9fd2dddfaef3b0c'>8c3c3b2fbc9cf08609ae30d5f9fd2dddfaef3b0c</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/menu-list/menu-list.spec.ts [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+    await element.press('ArrowDown');
+
+    await expect(menuItems.nth(2)).toBeFocused();
+
+    await element.press('ArrowDown');
+
+    await expect(menuItems.nth(3)).toBeFocused();
+  });
+
+  test('should close the menu when pressing the escape key', async () => {
```
Comment (halerankin) [inline packages/web-components/src/menu-list/menu-list.spec.ts:360]: Looks like this is testing to see if the focus returns to the parent item after the Esc key is pressed. I would reword the test to make your assertion clearer.

File: packages/web-components/src/menu-list/menu-list.spec.ts [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+    await expect(menuItems.nth(1)).toHaveAttribute('data-indent', '1');
+    await expect(menuItems.nth(2)).toHaveAttribute('data-indent', '1');
+    await expect(menuItems.nth(3)).toHaveAttribute('data-indent', '1');
+  });
+
+  test('should set the data-indent attribute to 1 correctly on all MenuItem elements when a menuitem in the menu as a role of menuitemradio', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+                <fluent-menu-list>
+                    <fluent-menu-item role="menuitemcheckbox"></fluent-menu-item>
```
Comment (halerankin) [inline packages/web-components/src/menu-list/menu-list.spec.ts:495]: menuitemradio

**Review by halerankin — APPROVED** [review]
w/ comment

File: packages/web-components/src/menu-list/menu-list.spec.ts (hunk 1 above)
Comment (brianchristopherbrady) [reply packages/web-components/src/menu-list/menu-list.spec.ts:360]: good catch

File: packages/web-components/src/menu-list/menu-list.spec.ts (hunk 2 above)
Comment (brianchristopherbrady) [reply packages/web-components/src/menu-list/menu-list.spec.ts:495]: good catch!

**Review by miroslavstastny — APPROVED** [review]
